.github/workflows/ci.yml

```diff
@@ -104,7 +104,7 @@ jobs:
           /bin/bash -c \
           '[ ! -f ${MJKEY_PATH} ] || mv ${MJKEY_PATH} ${MJKEY_PATH}.bak &&
           pytest --cov=garage --cov-report=xml -m \
-              'not nightly and not huge and not flaky and not large and not mujoco and not mujoco_long' --durations=20 &&
+              "not nightly and not huge and not flaky and not large and not mujoco and not mujoco_long" --durations=20 &&
```

Comment:
you only need single quotes where you are passing something to `/bin/bash -c`

Reply:
Oh I think using single quotes was ending the bash command, because it's working now

Reply:
Ahh right! Makes sense